spikes/aks-secure-baseline/deployments/spike/east/fluentbit.yaml

```diff
@@ -0,0 +1,19 @@
+---
```

Comment:
This file was copied from fluenbit helm chart in ngsa and modified.
- chart definition was updated to point to this spike
- container image was updated to pull from private container registry from aks-secure-baseline spike